<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Event Check-in Management</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #2c3e50; padding: 15px;</string>
      </property>
      <property name="text">
       <string>EVENT CHECK-IN MANAGERMENT</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::TabFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 170, 127);</string>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <widget class="QWidget" name="eventTab">
       <attribute name="title">
        <string>üìÖ Event Management</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btnAddEvent">
            <property name="styleSheet">
             <string notr="true">padding: 10px; font-size: 13px;</string>
            </property>
            <property name="text">
             <string>‚ûï New Event</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="eventTable">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(85, 170, 127);</string>
          </property>
          <property name="columnCount">
           <number>6</number>
          </property>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Event Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Date</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Place</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Description</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btnViewEvent">
            <property name="text">
             <string>üëÅ View Details</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnEditEvent">
            <property name="text">
             <string>‚úè Update</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDeleteEvent">
            <property name="styleSheet">
             <string notr="true">background-color: #e74c3c; color: white;</string>
            </property>
            <property name="text">
             <string>üóë Delete</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRefreshEvent">
            <property name="text">
             <string>üîÑ Refresh</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="attendeeTab">
       <attribute name="title">
        <string>üë• Attendee</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>üîç Search:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="attendeeSearch">
            <property name="placeholderText">
             <string>name, email, phone number, organisation, ...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="btnAddAttendee">
            <property name="styleSheet">
             <string notr="true">padding: 10px; font-size: 13px;</string>
            </property>
            <property name="text">
             <string>‚ûï Add Attendee</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="attendeeTable">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(85, 170, 127);</string>
          </property>
          <property name="columnCount">
           <number>6</number>
          </property>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Email</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Phone number</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Organisation</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Position</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="btnEditAttendee">
            <property name="text">
             <string>‚úè Update</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDeleteAttendee">
            <property name="styleSheet">
             <string notr="true">background-color: #e74c3c; color: white;</string>
            </property>
            <property name="text">
             <string>üóë Delete</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRefreshAttendee">
            <property name="text">
             <string>üîÑ Refresh</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="registrationTab">
       <attribute name="title">
        <string>üìù Registration</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Choose Event</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="eventCombo"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="btnRegisterAttendee">
            <property name="styleSheet">
             <string notr="true">padding: 10px; font-size: 13px;</string>
            </property>
            <property name="text">
             <string>‚ûï Register Attendee</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="registrationTable">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(85, 170, 127);</string>
          </property>
          <property name="columnCount">
           <number>6</number>
          </property>
          <column>
           <property name="text">
            <string>Code</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Email</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Organisation</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Regis Date</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Status</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QPushButton" name="btnGenerateQR">
            <property name="text">
             <string>üì±QR</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnCancelRegistration">
            <property name="styleSheet">
             <string notr="true">background-color: #e74c3c; color: white;</string>
            </property>
            <property name="text">
             <string>‚ùå Delete </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRefreshRegistration">
            <property name="text">
             <string>üîÑ Refresh</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="checkinTab">
       <attribute name="title">
        <string>‚úÖ Check-in</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Choose Event</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="checkinEventCombo">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(85, 170, 127);</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="statsGroup">
          <property name="title">
           <string>üìä </string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Total:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="totalRegisteredLabel">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>24</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #3498db;</string>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Checked-in:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="totalCheckedinLabel">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>24</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #27ae60;
color: rgb(0, 0, 0);</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="checkinGroup">
          <property name="title">
           <string>‚úÖ Check-in</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Code / QR:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="checkinCode">
             <property name="placeholderText">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnCheckin">
             <property name="styleSheet">
              <string notr="true">background-color: #27ae60; color: white; padding: 10px; font-size: 14px;</string>
             </property>
             <property name="text">
              <string>‚úì Check-in</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>üìã Checked-in List :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="checkinTable">
          <property name="styleSheet">
           <string notr="true">alternate-background-color: rgb(85, 170, 127);
background-color: rgb(85, 170, 127);</string>
          </property>
          <property name="columnCount">
           <number>5</number>
          </property>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Email</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Organisation</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Check-in Date</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Code</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QPushButton" name="btnRefreshCheckin">
            <property name="text">
             <string>üîÑ Refresh</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
